demo/controllers/toolbarMenuController.ts

```diff
@@ -19,15 +19,24 @@
 
 export default class ToolbarMenuController {
   public toolbarMenu: any;
```

Comment:
Types for toolbarMenu and toolbarMenuView. 
That will help out others (like me ;)) with the correct IDE autocompletion.